### PR TITLE
Coding Standards #2

### LINKED ISSUE #2: Coding standards
Some comments regarding coding standards:
- Remove unnecessary code comments. Have comments only for explaining things that cannot be explained through code. (Further reading: [1](https://www.ufried.com/blog/comments_1/), [2](https://www.ufried.com/blog/comments_2/), [3](https://www.ufried.com/blog/comments_3/))
- Descriptive naming of variables: `product` is a better variable name than `prod`. Also, would recommend using camelCase for naming variables instead of names_with_underscores (this is what we follow in both fabric and consult-api)
- Look for unused namespaces, imports and remove them. Should never commit any unused/unreachable code

### PR DESCRIPTION
**Pull request by sauravp1**
Why this is needed:
     to maintain readability and clarity.

Changes:

changed variable name from snake_case to camelCase.
Changed variable name from prod to product and ord to order.
removed all unused namespaces.

### CODE REVIEW

File: src/Entity/Product.php [hunk 1]
```diff
@@ -88,27 +87,27 @@ public function setPrice(string $price): self
...
+        if (!$this->product->contains($product)) {
+            $this->product[] = $product;
+            $product->setProductId($this);
         }
 
         return $this;
     }
 
-    public function removeProd(Orders $prod): self
+    public function removeProd(Orders $product): self
```
Comment (srinidhibhat-practo) [inline src/Entity/Product.php]: Method name can be renamed to `removeProduct`

File: src/Repository/OrdersRepository.php [hunk 2]
```diff
@@ -76,12 +49,7 @@ public function removeOrder(Orders $order)
 
     public function updateOrder($order, $new_data) 
     {
-        // $customer_id = $data["customer_id"];
-        // $product_id = $data["product_id"];
-        // $price = $data["price"];
-        // $customer = $this->customerRepository->findOneBy(['id'=>$customer_id]);
-        // $product = $this->productRepository->findOneBy(['id'=>$product_id]);
-
+
```
Comment (srinidhibhat-practo) [inline src/Repository/OrdersRepository.php]: Remove blank lines

**Review by srinidhibhat-practo — COMMENTED** [review]
Good job in cleaning up the code and refactoring the naming and usages 👍

File: src/Entity/Product.php (hunk 1 above)
Comment (sauravp1) [reply src/Entity/Product.php]: renamed method name

File: src/Repository/OrdersRepository.php (hunk 2 above)
Comment (sauravp1) [reply src/Repository/OrdersRepository.php]: removed blank lines

**Review by srinidhibhat-practo — APPROVED** [review]

**Pull request merged**